src/lib/swapper/swappers/ZrxSwapper/zrxBuildTrade/zrxBuildTrade.ts

```diff
@@ -63,8 +68,10 @@ export async function zrxBuildTrade<T extends ZrxSupportedChainId>(
       sellAmount: normalizeAmount(sellAmount),
       takerAddress: receiveAddress,
       slippagePercentage: slippage ? bnOrZero(slippage).toString() : DEFAULT_SLIPPAGE,
-      affiliateAddress: AFFILIATE_ADDRESS,
+      affiliateAddress: AFFILIATE_ADDRESS, // Used for 0x analytics
```

Comment:
this is the fox contract address. shoudl we be using this or the eth treasury address? 1inch swapper is using eth treasury address so maybe we should make this consistent

Reply:
I'm hesitant to change this at this juncture, as we've always used this for analytics (it populates a dashboard on their side).